client/dashboard/profile-wizard/style.scss

```diff
@@ -140,3 +141,26 @@
 		}
 	}
 }
+
+.woocommerce-profile-wizard__header.is-stepper {
+	display: block;
+
+	svg > g {
+		transform: initial;
+	}
+
+	@include breakpoint( '<782px' ) {
+		position: absolute;
```

Comment:
Should this be fixed to the bottom of the screen or sitting at the bottom of the page in mobile?  Most page content will push this below the fold, but not sure if a fixed item on the screen will affect usability. /cc @jameskoster